pkg/cmd/roachtest/backup.go

```diff
@@ -18,9 +18,22 @@ package main
 import (
 	"context"
 	"fmt"
-	"sync"
+
+	"golang.org/x/sync/errgroup"
 )
 
+func downloadStoreDumps(ctx context.Context, c *cluster, location string, binVersion string, nodeCount int) error {
```

Comment:
Eagle eyes please, @petermattis. Don't want to break this again. FWIW, I've run my in-progress test that uses this and it worked.